qemu/tests/qemu_guest_agent.py

```diff
@@ -146,8 +146,8 @@ def _check_ga_pkg(self, session, cmd_check_pkg):
                                 "of qga rpm in advance" % self.qga_pkg_path)
             elif version_list[1] != version_list[0]:
                 return False
-        elif self.params.get("os_variant", "") == 'rhel7':
-            pattern = r"guest-agent-(\d+.\d+.\d+-\d+).el7"
+        elif self.params.get("os_variant", "") in ["rhel7", "rhel9"]:
```

Comment:
```suggestion
        else:
```
Since currently all releases besides `rhel8` all go here, if other more releases in the future keep this, you won't need add them one by one.

Reply:
Yeah, I thought about the previous versions, but apparently, it's enough. thanks for your suggestions. here is the result.
(1/1) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.qemu_guest_agent.virtio_serial.check_guest_info.q35: PASS (32.62 s)